LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-valid.html

```diff
@@ -22,14 +22,14 @@
 test_valid_value("animation", "reverse", ["reverse", "0s ease 0s 1 reverse none running none"]);
 test_valid_value("animation", "both", ["both", "0s ease 0s 1 normal both running none"]);
 test_valid_value("animation", "paused", ["paused", "0s ease 0s 1 normal none paused none"]);
-test_valid_value("animation", "none", ["none", "0s ease 0s 1 normal none running none"]);
+test_valid_value("animation", "none", ["0s", "none", "0s ease 0s 1 normal none running none"]);
```

Comment:
Perhaps a personal preference, but I have a bad feeling about using `0s` as serialization here. I could see web content checking that `none` is contained in the serialization.

Reply:
I am 100% fine sticking with "none" here, but what is the rule that follows? Would you like me to just code and exception for this case or do you have a rule to propose?

The rule I was following was to serialize the initial value of the first longhand in this case.

Reply:
https://www.w3.org/TR/cssom-1/#serializing-css-values
> If either of the above syntactic translations would be less backwards-compatible, do not perform them.

I admit I haven't done any real research on that bit, but I wouldn't be surprised if a regression came up because some animation library or website searched for `none` in the serialization. @karlcow Do you know if serializing to `animation: 0s` instead of `animation: none` could cause an issue ?

Anyway, I'm not blocking on this, but something to think about.

Reply:
I would like to choose whether to switch to `0s` or not. I am happy to code it as a special case for website compatibility, real or imagined, or to invent some rule (not sure what the rule would be). I understand your compatibility worry, but I will note that many of the CSS serialization changes we are making come with compatibility risk, and to me this particular one doesn’t seem greater than the others.

Reply:
Are we talking about this case?

```
data:text/html,<div%20style="animation:none">
```

So right now.
WebKit, Gecko and Blink all return:

```
0s ease 0s 1 normal none running none
```

I checked a bit. I don't know about any issues. Specifically I see people assigning the value to none, but I didn't find **yet**, people testing against  `none` for the matter. 

Example: 

https://github.com/ashwinexe/Prehackstoric-Hack/blob/5bcd76f39489a450d12f3b36385aa2df76b8573f/scripts/game.js#L70

0 occurrence of `none ease 0s 1 normal none running none`
272 occurrences of `0s ease 0s 1 normal none running none`

Assuming I got the framing right at the start of this comment.

Reply:
Thanks Karl, you got it exactly right.

Reply:
@karlcow I was wondering if they do checks like `style.animation.includes("none")` or such. I couldn't find anything from a quick Github search, so we might be safe here :) Checks like the one you found are also problematic, but we might just have to deal with it.